Makefile

```
@@ -799,10 +799,13 @@ set-manifest-image:
 ##@ vm-operator:
 
 .PHONY: release-vm-operator
-release-vm-operator: docker-vm-operator-build-all vm-operator-manifest-build docker-vm-operator-push-all ## Build and push the vm-operator image and manifest for usage in CI
+release-vm-operator: docker-build-all-vm-operator generate-manifests-vm-operator docker-push-all-vm-operator clean-vm-operator ## Build and push the vm-operator image and manifest for usage in CI
 
-.PHONY: vm-operator-checkout
-vm-operator-checkout:
+.PHONY: release-local-vm-operator
+release-local-vm-operator: docker-build-all-vm-operator generate-manifests-vm-operator clean-vm-operator ## Build and push the vm-operator image and manifest for local usage only
```

```suggestion
release-local-vm-operator: docker-build-all-vm-operator generate-manifests-vm-operator clean-vm-operator ## Build the vm-operator image and manifest for local usage only
```
The diff seems to be that we don't push.

Q: Do you want to name this target release-vm-operator-local? (no strong opinion from my side)